treeherder/model/derived/jobs.py

```diff
@@ -1937,15 +1937,24 @@ def _load_log_urls(self, log_placeholders, job_id_lookup,
                 log_placeholders[index][0] = job_id
                 log_placeholders[index].append(time_now)
                 task = dict()
-                task['job_guid'] = job_guid
-                task['log_url'] = log_ref[2]
-                task['result_set_id'] = result_set_id
-                task['check_errors'] = True
-                if result != 'success':
-                    task['routing_key'] = 'parse_log.failures'
+
+                if log_ref[1] == 'structured-raw':
+                    # don't parse structured logs for passing tests
```

Comment:
Yeah, I opted initially to only parse failed tests.  I figured we could easily add parsing for passing later by taking out the check.  But, right.  The perf hit could be substantial.  We may need to experiment with that when we cross that bridge.